cityofaustin/atd-vz-data
# Update LocationMap

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/15867

**Please review these PRs in this order: https://github.com/cityofaustin/atd-vz-data/pull/1398, https://github.com/cityofaustin/atd-vz-data/pull/1399, https://github.com/cityofaustin/atd-vz-data/pull/1400**

This PR reimplements the `LocationMap` to use `react-map-gl` v7 and prevents map zoom from happening on page scroll.

## Testing
**URL to test:** 
Local

**Steps to test:**
1. FYI you will need to run `npm i` in the editor folder when switching back and forth between this branch and others without the map library update
2. Go to a location page and make sure the map loads
3. See that there are zoom and fullscreen controls in the top left just like in staging and production
4. Try scrolling while hovering the mouse over the map. You should see a prompt to use a cooperative gesture to zoom the map in and out.
5. Notice that the location polygon is outlined with yellow just like in staging and production. There is also the same amount of padding set around the polygon and the edges of the map.

---
#### Ship list
